assets/app/scripts/controllers/settings.js

```diff
@@ -34,12 +34,19 @@ angular.module('openshiftConsole')
         angular.forEach(limitRange.spec.limits, function(limit) {
           limit.min = limit.min || {};
           limit.max = limit.max || {};
+          limit.default = limit.default || {};
           angular.forEach(limit.max, function(value, type) {
             limit.min[type] = limit.min[type] || "";
+            limit.default[type] = limit.default[type] || "";
```

Comment:
Firefox didn't seem to mind since it was a key on an object and not a top level var.  Didn't test all the other browsers though.  Going ahead and switching it just in case.